spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -20,20 +24,38 @@ def it_redirects_to_user_login
         get :new
         expect(response).to render_template("new")
       end
+
+      it "with chapter" do
```

Comment:
This looks like a `context` description, not a description of the expected behavior. Line 47 also has the same issue.

Reply:
Fixed